src/_P088_Ping.ino

```diff
@@ -31,7 +31,7 @@ extern "C"
 
 struct P088_icmp_pcb {
   struct raw_pcb *m_IcmpPCB = NULL;
-  uint8_t instances;
+  uint8_t instances; /* Sort of refcount */
```

Comment:
This one is not initialized to 0.
And it is good practice to use `nullptr` instead of `NULL`.
The compiler can check `nullptr` for a pointer type, but cannot perform such checks for `NULL`.

If it should have a certain value at creation, then assigning such value should be done in the constructor.

```c++
P088_icmp_pcb() : m_IcmpPCB(nullptr), instances(1) {}
```

Reply:
Will do this now, but i am not aware how lwip will react to nullptr, as it is not C++.